docs/src/user/algorithms.rst

```diff
@@ -250,6 +251,11 @@ True to set equal weights for observed points. Default is ``False``.
 
 The weight given to the prior point of the input space. Default is ``1.0``.
 
+``full_weight_num``
+
+The number of the most recent trials which get the full weight where the others will be
+applied with a linear ramp from 0 to 1.0. It will only take effect if ``equal_weight``
+is ``False``. Default is ``25``.
```

Comment:
This is consistent with the paper but not with the implementation, right?

Reply:
I suddenly realize seems I misunderstand this ramp up when you said from 1/n to 1. Paper mentions ramp from 0 to 1.0, but the weight of course can not be 0, so get start from 1/n is not that different compared with the paper.

Reply:
But it's (1/n, 1/n, ... 1/n,  growing to 1 through 25 steps), while paper says (0 growing to 1, 1, 1, ... 25 times).